From 11e44a5ed8454481f4c11ca9c2980de021f5b069 Mon Sep 17 00:00:00 2001
From: Rico Ganahl <rico.ganahl@bytesatwork.ch>
Date: Wed, 28 Jun 2023 12:04:54 +0200
Subject: [PATCH] bytesatwork: bawconfig: ti am62x ram fallback

Add fallback RAM for TI AM62x on byteDEVKIT.

Signed-off-by: Rico Ganahl <rico.ganahl@bytesatwork.ch>
(cherry picked from commit 1ded65d63973f6553ac83dd0f2fa5b3010ef3872)
---
 board/bytesatwork/baw_config/baw_config_get.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/board/bytesatwork/baw_config/baw_config_get.c b/board/bytesatwork/baw_config/baw_config_get.c
index 86ccdb66a8..bf171fa39e 100644
--- a/board/bytesatwork/baw_config/baw_config_get.c
+++ b/board/bytesatwork/baw_config/baw_config_get.c
@@ -87,6 +87,8 @@ default_config:
 			config->ram = M5_RAM_K4B4G1646DBIK0;
 		else if (IS_ENABLED(CONFIG_TARGET_IMX8MM_BYTEDEVKIT))
 			config->ram = M6_RAM_MT53E128M32D2DS_053;
+		else if (IS_ENABLED(CONFIG_TARGET_IMX8MM_BYTEDEVKIT))
+			config->ram = M8_RAM_MT53E256M16D1DS_046;
 	}
 
 	if (IS_ENABLED(CONFIG_SPL_BUILD)) {
-- 
2.39.2

